### DISCUSSION TITLE
Is it not possible to install nchat on Manjaro Linux?

### DISCUSSION THREAD

**Discussion by kelvinauta** [d99kris/nchat#309] [Q&A]
I tried 3 ways to install: nchat and nchat-git from AUR; and also manually,  
but in all methods I get an error. 

### Error in aur install
```
cmake: error while loading shared libraries: libjsoncpp.so.25: cannot open shared object file: No such file or directory
==> ERROR: Se produjo un fallo en build().                         A
    Cancelando...
 -> error compilando: nchat-exit status 4                          D
comprobando dependencias...
ft  Speed
Paquetes (1) gperf-3.1-5
   0      0 --:--:-- --:--:-- --: 
Tamaño total quitado:  0,22 MiB
  0      0 --:--:-- --:--:-- --:  
```

### error in manual installl:
```
   24 |   struct WaitFreeStorage {
      |          ^~~~~~~~~~~~~~~
0x21a7bea internal_error(char const*, ...)
	???:0
0xfd061b split_critical_edges(bool)
	???:0
0x11dfc41 tail_merge_optimize(bool)
	???:0
Please submit a full bug report, with preprocessed source (by using -freport-bug).
```

**Comment by d99kris** [comment]
I think nchat should work on Manjaro Linux, I recall other users mentioned using it. I could give it a shot here. Can you share what version (24.0.0?) and flavour (Plasma, Xfce?) of Manjaro you're on, and what CPU arch (e.g. x86-64)? Also the amount of system RAM would be useful to know. Thanks!

**Comment by d99kris** [comment]
I will proceed to close this discussion for now, but feel free to re-open it once/if you have the information requested. Also knowing the commands used to build / install nchat would be useful when debugging. Thanks!
